worker/caasoperator/caasoperator.go

```diff
@@ -136,6 +168,15 @@ func (op *caasOperator) loop() (err error) {
 		)
 	}
 
+	// First things, run the config changed hook.
```

Comment:
Move this to the TODO(axw) below. The watcher will be triggered once at startup.

Reply:
moved